api/server.js

```diff
@@ -0,0 +1,40 @@
+// Setting requirements for the server file
+const express = require('express');
+const helmet = require('helmet');
+const cors = require('cors');
+
+// Setting imports for the server file (Commenting out so nothing breaks... yet)
+
+const dishRouter = require('../components/dish/dish-router.js');
+const ingredientRouter = require('../components/ingredients/ingredient-router.js');
+const measurementRouter = require('../components/measurement/measurement-router.js');
```

Comment:
I would also add that because they're all under the api route, they should all be inside the api folder. So you don't really need a components folder. I will show you the file structure next week.